content/ja/docs/reference/glossary/persistent-volume-claim.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 永続ボリューム要求
```

Comment:
PersistentVolumeClaim

Reply:
原文だと `Persistent Volume Claim` となっているので `永続ボリューム要求` のままにすべきかどうか

Reply:
スペース入っちゃっているやつですよね
訳しちゃっていいかなと思います

あと
`Claims storage resources defined in a PersistentVolume so that it can be mounted as a volume in a container.`
の訳ですが、ちょっと機械ぽいので

コンテナ内でボリュームとしてマウントするためのPersistentVolume内で定義されたストレージリソースを要求します。

とかでどうでしょうか (`ために`でもいいかも)